addons/source-python/docs/source-python/source/developing/module_tutorials/menus.rst

```diff
@@ -0,0 +1,117 @@
+Menus
+======
+
+This page contains tutorials about the :mod:`menus` package.
+
+
+Creating PagedMenu
+--------------------------
+
+This is a simple example to create paged menu
+
+.. code-block:: python
+
+    import random
+
+    from commands.say import SayCommand
+
+    from menus import PagedMenu
+    from menus import PagedOption
+
+    from players.entity import Player
+
+
+    # Register menu command
+    @SayCommand(['menu', '/menu', '!menu'])
+    def say_command(command, index, teamonly):
+        # Send the menu
+        menu.send(index)
+        return False
+
+    menu = PagedMenu(title='Welcome menu', description='Choose an option:', select_callback=my_select_callback)
+
+    # Add options from 1 to 20
+    for i in range(1, 20):
+        menu.append(PagedOption(f'{i}', i))
```

Comment:
This should be declared after `my_select_callback` because it currently won't load and produce a `NameError`. Moreover, the first line is too long and does not conform with [PEP 8](https://peps.python.org/pep-0008/), which we try to follow and promote for this project.